mllam_data_prep/ops/cropping.py

```diff
@@ -0,0 +1,201 @@
+import numpy as np
+import spherical_geometry as sg
+import xarray as xr
+from spherical_geometry.polygon import SphericalPolygon
+
+
+def _get_latlon_coords(da: xr.DataArray) -> tuple:
+    """
+    Get the latlon coordinates of a DataArray.
+
+    Parameters
+    ----------
+    da : xarray.DataArray
+        The data.
+
+    Returns
+    -------
+    tuple (xarray.DataArray, xarray.DataArray)
+        The latitude and longitude coordinates.
+    """
+    if "latitude" in da.coords and "longitude" in da.coords:
+        return (da.latitude, da.longitude)
+    elif "lat" in da.coords and "lon" in da.coords:
+        return (da.lat, da.lon)
+    else:
+        import ipdb
```

Comment:
ipdb is now only a dev-dependency, but if it should be used here it needs to be a general dependency (or latlon-domain-crop)